src/server/game/Chat/Channels/Channel.h

```diff
@@ -148,106 +146,121 @@ class TC_GAME_API Channel
+        void MakePlayerNotFound(WorldPacket* data, std::string const& name) const;          //+ 0x09
+        void MakeNotOwner(WorldPacket* data) const;                                         //? 0x0A
+        void MakeChannelOwner(WorldPacket* data) const;                                     //? 0x0B
+        void MakeModeChange(WorldPacket* data, ObjectGuid guid, uint8 oldflags) const;      //+ 0x0C
+        void MakeAnnouncementsOn(WorldPacket* data, ObjectGuid guid) const;                 //+ 0x0D
+        void MakeAnnouncementsOff(WorldPacket* data, ObjectGuid guid) const;                //+ 0x0E
+        void MakeMuted(WorldPacket* data) const;                                            //? 0x11
+        void MakePlayerKicked(WorldPacket* data, ObjectGuid bad, ObjectGuid good) const;    //? 0x12
+        void MakeBanned(WorldPacket* data) const;                                           //? 0x13
+        void MakePlayerBanned(WorldPacket* data, ObjectGuid bad, ObjectGuid good) const;    //? 0x14
+        void MakePlayerUnbanned(WorldPacket* data, ObjectGuid bad, ObjectGuid good) const;  //? 0x15
+        void MakePlayerNotBanned(WorldPacket* data, std::string const& name) const;         //? 0x16
+        void MakePlayerAlreadyMember(WorldPacket* data, ObjectGuid guid) const;             //+ 0x17
+        void MakeInvite(WorldPacket* data, ObjectGuid guid) const;                          //? 0x18
+        void MakeInviteWrongFaction(WorldPacket* data) const;                               //? 0x19
+        void MakeWrongFaction(WorldPacket* data) const;                                     //? 0x1A
+        void MakeInvalidName(WorldPacket* data) const;                                      //? 0x1B
+        void MakeNotModerated(WorldPacket* data) const;                                     //? 0x1C
+        void MakePlayerInvited(WorldPacket* data, std::string const& name) const;           //+ 0x1D
+        void MakePlayerInviteBanned(WorldPacket* data, std::string const& name) const;      //? 0x1E
+        void MakeThrottled(WorldPacket* data) const;                                        //? 0x1F
+        void MakeNotInArea(WorldPacket* data) const;                                        //? 0x20
+        void MakeNotInLfg(WorldPacket* data) const;                                         //? 0x21
+        void MakeVoiceOn(WorldPacket* data, ObjectGuid guid) const;                         //+ 0x22
+        void MakeVoiceOff(WorldPacket* data, ObjectGuid guid) const;                        //+ 0x23
+
+        void SendToAll(WorldPacket* data, ObjectGuid guid = ObjectGuid::Empty) const;
+        void SendToAllButOne(WorldPacket* data, ObjectGuid who) const;
+        void SendToOne(WorldPacket* data, ObjectGuid who) const;
+
+        bool IsOn(ObjectGuid who) const { return _playersStore.count(who) != 0; }
+        bool IsBanned(ObjectGuid guid) const { return _bannedStore.count(guid) != 0; }
```

Comment:
Isn't using count considered a bad practice? Guess it doesnt matter really, but just thought it was weird that this was changed for no real reason.